src/ffishjs.cpp

```diff
@@ -562,10 +562,14 @@ Game read_game_pgn(std::string pgn) {
         headersParsed = true;
         auto it = game.header.find("Variant");
         if (it != game.header.end()) {
-          game.variant = it->second;
+          game.is960 = it->second.find("960") != std::string::npos;
+          if (game.is960) {
+            game.variant = it->second.substr(0, it->second.length() - 3);
```

Comment:
Either the `find` should be restricted to start searching at `size() - 3` or it should erase starting from the `find` position in order to ensure that 960s occuring elsewhere in the string won't lead to weird artifacts like `960chess` -> `960ch`.